src/components/Search/SelectedRevisionsTable/SelectedRevisionsTable.jsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+
+import Close from '@mui/icons-material/Close';
+import { TableRow } from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import PropTypes from 'prop-types';
+import { connect, useDispatch } from 'react-redux';
+
+import { deleteRevision } from '../../../reducers/SelectedRevisions';
+import { Alignment } from '../../Shared/enums';
+import StyledTableCell from '../../Shared/StyledTableCell';
+import StyledTableRow from '../../Shared/StyledTableRow';
+import './SelectedRevisionsTable.module.css';
```

Comment:
css imports not fully functional yet, waiting on probably a PR that has webpack config before I can add the css loaders in the webpack..